packages/block-editor/src/layouts/flex.js

```diff
@@ -44,12 +44,15 @@ const alignItemsMap = {
 	left: 'flex-start',
 	right: 'flex-end',
 	center: 'center',
+	stretch: 'stretch',
 };
 
 const verticalAlignmentMap = {
 	top: 'flex-start',
 	center: 'center',
 	bottom: 'flex-end',
+	stretch: 'stretch',
+	spaceBetween: 'space-between',
```

Comment:
Totally not urgent, but the horizontal alignment uses the kebab case to match the CSS value, rather than the camelCase JS version of space between. Should we make these consistent and use `space-between` here, too, for the value?

Reply:
Done!